README.md

```
@@ -1,44 +1,417 @@
 #### Installation
+
 ```
-npm install node-binance-api --save
+npm install node-binance-api
+```
+[![NPM](https://nodei.co/npm/node-binance-api.png?compact=true)](https://npmjs.org/package/node-binance-api)
+<!-- [![npm install node-binance-api](https://nodei.co/npm/node-binance-api.png?mini=true)](https://npmjs.org/package/node-binance-api) -->
+
+#### Community Telegram Chat
+
+https://t.me/nodebinanceapi
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)